content/docs/docs-nav.yaml

```diff
@@ -6,23 +6,24 @@ plugins:
 
 nav:
   - Getting started:
-    - Initial configuration: '/docs/getting-started/getting-started-for-admins/'
+    - Installing the GitHub app: '/docs/getting-started/install-github-app/'
+    - Designating admins: '/docs/getting-started/create-admin-group/'
```

Comment:
I wonder should this be in its own section with inviting users and called it "User Management"?

Reply:
The idea is that the Getting Started docs are in order. My thought was that after installing the GitHub app, making an admin is the next most important thing for the user to do.

However, now that I think about it, if they can add components etc. without making themselves an admin (because they are an admin by default), then you are right. The instructions for making admins are not useful until they want to add a new user.

Is that what you're thinking?

Reply:
Yes.

Reply:
Yeah good idea. I've done the following...

1. Move "Designating admins" after "Inviting users" in the sidebar. This should mean they can invite some users who will all be admins and then learn how to decide who is and isn't an admin afterwards.
2. Link to "Designating admins" in the Next Steps section of the "Inviting users doc".

Does that make more sense now?

Reply:
yes.